octopus/lib/octopus_web/live/manager_live.ex

```diff
@@ -158,8 +160,11 @@ defmodule OctopusWeb.ManagerLive do
             canvas -> Canvas.to_svg(canvas, width: "100%", height: "100%")
           end
 
-        %{module: module, name: name, icon: icon}
+        category = apply(module, :category, [])
+
+        %{module: module, name: name, icon: icon, category: category}
       end
+      |> Enum.group_by(& &1.category)
```

Comment:
Maybe we could sort the apps by group in a way that test apps are not at the top.
Something like `|> Enum.sort_by(&Map.get(%{:animation: 0, game: 1, test: 2, ... and so on}, &1.category))`